lib/mdl.rb

```diff
@@ -107,6 +107,30 @@ def self.run(argv=ARGV)
     if Config[:json]
       require 'json'
       puts JSON.generate(results)
+    elsif Config[:junit]
+      output = ""
```

Comment:
I don't love the idea of hand-creating XML.... what do you think of using something like `builder` instead?

Since it's relatively simple XML, I'm not absolutely ruling it out, but I'd prefer to explore a library that handles this for us and is robust.

Reply:
> library that handles this for us and is robust

I think there's something to learn from https://github.com/rubocop/rubocop/blob/master/lib/rubocop/formatter/formatter_set.rb

Especially:

https://github.com/rubocop/rubocop/blob/fe2d7ef59051f11f52bfbf421aa96d69bdc5b2d7/lib/rubocop/formatter/junit_formatter.rb#L55-L62

and

https://github.com/rubocop/rubocop/blob/fe2d7ef59051f11f52bfbf421aa96d69bdc5b2d7/lib/rubocop/formatter/junit_formatter.rb#L80-L90

Reply:
This is more succinct: https://github.com/mikian/rubocop-junit-formatter/blob/v0.1.4/lib/rubocop/formatter/junit_formatter.rb